src/api/kube.ts

```diff
@@ -759,6 +759,41 @@ export class KubeHelper {
     }
   }
 
+  readClassStorage(yamlPath: string): V1Deployment {
```

Comment:
The name is n't consistent with a function.
The function reads deployment object.